get-started/validators.mdx

```diff
@@ -40,7 +40,9 @@ By opting-in to the mev-commit protocol as a Holesky validator, you agree to the
   </tbody>
 </table>
 
-If you are a relay looking to join mev-commit network please visit our [Relays page](/get-started/relays) to get more information.
+If you are a relay looking to join the mev-commit network please visit our [Relays page](/get-started/relays) to get more information.
+
+Proposers who fallback to local block production, from either a fault in the mev-boost software or a lack of economically viable block bids, **will not be slashed**. See [flashbots docs](https://docs.flashbots.net/flashbots-mev-boost/architecture-overview/risks#liveness-and-local-fallback) for more on this scenario. The protocol may still slash proposers who abuse this local fallback mechanism.
```

Comment:
"The protocol may still slash proposers who abuse this local fallback mechanism."
It currently does not do that, so seems a bit weird to say this.

Reply:
It's more saying we reserve the right to add additional mechanisms (or manually slash) any proposer who abuses local fallback

Reply:
thanks for bringing that into consideration guys. we can leave it out for now, I'll do that & merge

Reply:
The protocol cannot manually slash. Maybe clearer to say "The protocol may in the future still slash proposers who abuse this local fallback mechanism." or something like that.

Reply:
Manual as in social slashing. There's no reason this isn't possible, and is a feature of the eigenlayer token. But I see the argument that we shouldn't go down this route